nullaway/src/test/java/com/uber/nullaway/CoreTests.java

```diff
@@ -207,7 +207,7 @@ public void cfNullableArrayField() {
   }
 
   @Test
-  public void supportSwitchExpression() {
+  public void switchOnNullable() {
```

Comment:
Rename to clarify what is being tested; this test was written before Java added `switch` expressions